adam-core/src/test/scala/org/bdgenomics/adam/util/ADAMFunSuite.scala

```diff
@@ -25,6 +25,7 @@ trait ADAMFunSuite extends SparkFunSuite {
   override val properties: Map[String, String] = Map(("spark.serializer", "org.apache.spark.serializer.KryoSerializer"),
     ("spark.kryo.registrator", "org.bdgenomics.adam.serialization.ADAMKryoRegistrator"),
     ("spark.kryoserializer.buffer.mb", "4"),
-    ("spark.kryo.referenceTracking", "true"))
+    ("spark.kryo.referenceTracking", "true"),
+    ("spark.hadoop.parquet.task.side.metadata", "false"))
```

Comment:
We can't use `parquet.task.side.metadata` with the `filter2` API?  Or just better to not use it in general?